### PR TITLE
feature #17: Added isAlive endpoint. Tests added.

### LINKED ISSUE #17: Feature: Add endpoint /alive for checking if the storage engine is working
### Current problem

Implement /alive endpoint according [server API documentation](https://reduct.store/docs/http-api/server-api) 

### Possible solution

_No response_

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Rumpelshtinskiy**
Closes #17 

### Please check if the PR fulfills these requirements

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [x] CHANGELOG.md has been updated (for bug fixes / features / docs)

### Changes

Added isAlive endpoint. Tests added. Successful example for testing a server info enpoint was changed - replace strings values to inegers.

### New behavior

Now an interface Server Client present an isAlive method, which implements api/v1/alive endpoint for checking if the storage engine is working.

### CODE REVIEW

File: src/test/java/org/reduct/client/util/ServerClientConstants.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package org.reduct.client.util;
 
-public class ServerInfoConstants {
+public class ServerClientConstants {
```
Comment (atimin) [inline src/test/java/org/reduct/client/util/ServerClientConstants.java:3]: `ResponseExamples` sounds less confusing.

Comment (atimin) [reply src/test/java/org/reduct/client/util/ServerClientConstants.java:3]: Could you also check SUCCESSFUL_SERVER_INFO_RESPONSE?  The old C++ implementation used to return integers as strings, now it is normal integers. I'm wondering if it breaks something in Java.

**Review by atimin — CHANGES_REQUESTED** [review]
Please, fill the PR template.

File: src/test/java/org/reduct/client/util/ServerClientConstants.java (hunk 1 above)
Comment (Rumpelshtinskiy) [reply src/test/java/org/reduct/client/util/ServerClientConstants.java]: The ServerClientConstants renamed. In SUCCESSFUL_SERVER_INFO_RESPONSE changes all strings to integers and it was tasted successfuly.

**Comment by atimin** [comment]
@Rumpelshtinskiy , thank you for you contribution!

**Pull request merged**